### PR TITLE
Issue-1 Replace XD with Data Flow

### PR DESCRIPTION
**Pull request by ghillert**
* Replace job with task references
* Fix minor Task module listing
* Fix Stream listing

This closes spring-cloud/spring-cloud-dataflow-ui#1
This closes spring-cloud/spring-cloud-dataflow-ui#4
This closes spring-cloud/spring-cloud-dataflow-ui#13

### CODE REVIEW

File: ui/app/index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@
                <a ui-sref="home.streams.tabs.definitions">Streams</a>
             </li>
             <li ng-class="{ active: $state.includes('home.jobs') }">
```
Comment (ilayaperumalg) [inline ui/app/index.html]: Would be good to replace the variable name `job(s)` to `task(s)` for consistency.

Comment (ilayaperumalg) [inline ui/app/index.html]: @ghillert I see this in several places now. May be it is good to have `task` in all the places (including the package/file names) given `task` is a superset now. What do you think?

**Comment by ilayaperumalg** [comment]
I believe this PR needs to be updated to revert the changes on renaming `task` -> `job`. @ghillert Could you confirm?

**Comment by ghillert** [comment]
Added `Runtime Modules` tab

**Comment by ilayaperumalg** [comment]
LGTM, squashed, rebased and merged as 0e65e577dc70ad72e2427cd8f6ef377cc68af596

**Pull request closed without merging**